sing in.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+    
+  

```

Comment:
### Title tag is missing
Please put in a title tag with descriptive title.

[More about importance of title tag](https://moz.com/learn/seo/title-tag#:~:text=Why%20are%20title%20tags%20important,your%20page%20via%20organic%20search.)